vars/main.yml

```diff
@@ -77,9 +77,9 @@ default_monitor_config:
 default_mdb_config:
   # General
   olcDatabase: "{2}mdb"
-  olcDbDirectory: "{{ db_dir | mandatory }}"
+  olcDbDirectory: "{{ slapd_mdb_dir | mandatory }}"
```

Comment:
please mention and describe mandatory vars by name in readme. Also you forgot to replace usages of `db_dir` in readme.